packages/db/prisma/schema.prisma

```diff
@@ -52,11 +53,18 @@ model Activity {
 }
 
 model Absence {
-  userId      Int
-  user        User     @relation(fields: [userId], references: [id])
-  absenceDate DateTime @db.Date
+  participant   Participant @relation(fields: participantId, references: id)
+  participantId Int
+  absenceDate   DateTime    @db.Date
 
-  @@unique([absenceDate, userId])
+  @@unique([absenceDate, participantId])
+}
+
+model Participant {
+  id       Int       @id @default(autoincrement())
```

Comment:
Can we have a comment here on why we can have multiple users mapped to one participant, while each User only maps to a singular participant?

Spoke to Baker and to my understanding the reason is that each User (parent) can have the same kid (Participant), so the state they see for the application should be the same, which is why we need this additional field to link them together.

Reply:
ok 👍 just a thought but what if a user (parent) had two participants (kids) in the program :o

Reply:
that isn't a case that the NPO has ever had historically -- we can build it out later if necessary, but this will be extra work so will come at the end of the project

Reply:
i see, tyty!